src/main/java/com/google/sps/data/DeleteAllFromDatastore.java

```diff
@@ -0,0 +1,28 @@
+package com.google.sps.data;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+
+// This class is to be used by the other three servlets to delete entities from datastore.
+public class DeleteAllFromDatastore {
```

Comment:
Change the name to utility class, so that you can add other shared methods into this class in the future.

Also you should put this method in the servlet package, and make it private to the package since this class should only contain utility methods for the servlets.

Reply:
Renamed the class.

Reply:
Sorry I didn't make this clear. I wanted the class name to exclude "delete from data store" thing so you can add other functions (that are not related to data store) to this class.

An example would be "ServletUtility" because it contains the utility methods for the servlet.

Reply:
Also you should make this class private to the package (cannot be accessed outside the package). Java has 4 visibility modifiers: https://docs.oracle.com/javase/tutorial/java/javaOO/accesscontrol.html

In your case, setting it to default (no modifier) would do.